### PR TITLE
Add jdcal recipe.

### PR DESCRIPTION
**Pull request by curtisalexander**

### CODE REVIEW

File: recipes/jdcal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - jdcal.is_leap
+    - jdcal.gcal2jd
+    - jdcal.jd2gcal
+    - jdcal.jcal2jd
+    - jdcal.jd2jcal
+    - jdcal.MJD_0
+
+about:
+  home: https://github.com/phn/jdcal 
+  license: BSD
```
Comment (jakirkham) [inline recipes/jdcal/meta.yaml]: Please specify license version (e.g. `BSD 2-Clause` for the 2-Clause version of the BSD license).

File: recipes/jdcal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - jdcal.ipart
+    - jdcal.is_leap
+    - jdcal.gcal2jd
+    - jdcal.jd2gcal
+    - jdcal.jcal2jd
+    - jdcal.jd2jcal
+    - jdcal.MJD_0
+
+about:
+  home: https://github.com/phn/jdcal
```
Comment (jakirkham) [inline recipes/jdcal/meta.yaml]: nit: Please remove the trailing whitespace on this line.

File: recipes/jdcal/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
```
Comment (jakirkham) [inline recipes/jdcal/meta.yaml]: Please remove the comment from the example above and this empty line.

**Comment by curtisalexander** [comment]
@jakirkham I made the changes recommended.  Thank you.

One more question - can you confirm that I setup the `test` section correctly?  Originally I had individual functions imported, such as `jdcal.fpart`.  You can see from my last `push` that I just import `jdcal`.

Thanks.

**Comment by mariusvniekerk** [comment]
@curtisalexander that is the correct way to test a simple python conda-forge recipe.

**Comment by curtisalexander** [comment]
@mariusvniekerk Thank you for clarifying.

**Comment by jakirkham** [comment]
Thanks @curtisalexander.

**Comment by jakirkham** [comment]
Also thanks @mariusvniekerk for helping out.

**Pull request merged**